mimicpy/core/_qmhelper.py

```diff
@@ -17,6 +17,9 @@ def cleanqdf(qdf):
     columns = mpt.columns.copy() # copy it otherwise original gets edited    
     columns.extend(['x','y','z'])
     col_to_drop = [l for l in qdf.columns if l not in columns]
+    
+    qdf.index = qdf.index.set_names(['id']) # rename index, so it is id when we reset index
```

Comment:
Where do you reset index?

Reply:
It is reset in prepare.getInp(). Line 95 of prepare.py.

I made the comment more clearer.